### ISSUE TITLE
UX/UI when user cannot access Military Information - text and accessibility issues

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#22589]
## What happened?
Text issue:  Missing the final period at the end of the paragraph.

Accessibility issue:  "We can't access your military information" is supposed to be marked as a header (header 2), but I'm not hearing any mention of header with VoiceOver/Talkback.  I always hear "header" when it is a header.

## Specs:
- Device: Android / iOS
Tested with user 1401.

## Steps to Reproduce
- Go to user who doesn't have military info available to them.

## Desired behavior
Has a final period.  Header is called "header" in TalkBack/Voiceover

## Acceptance Criteria
see above.

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/19521

## Screen shot(s) and additional information

Specs:

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/0d86f1f0-ac89-41bc-b717-be2bf21dbc12)

Actual:

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/a4b79bf0-7b09-4b36-8adc-ba1334e12fba)

**Comment by digitalken** [comment]
PASS 4/7/2021 - Looks good on Android and iOS.  Closing.
